src/misc/cache/lv_image_header_cache.h

```diff
@@ -0,0 +1,66 @@
+/**
+* @file lv_image_header_cache.h
+*
+ */
+
+#ifndef LV_IMAGE_HEADER_CACHE_H
+#define LV_IMAGE_HEADER_CACHE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_cache_private.h"
```

Comment:
Can this be moved to the .c file? https://github.com/lvgl/lvgl/issues/5899#issuecomment-2004582233

Reply:
`<stdbool.h>` will be needed here fyi, if you do move it.

Reply:
The LV_CACHE_DEF macro need be imported from this header

Reply:
Gotcha, thanks

Reply:
> The LV_CACHE_DEF macro need be imported from this header

Do you mean `LV_CACHE_DEF_SIZE`? It should be imported by lv_conf_internal.h. Let's keep the private headers private.

Reply:
You're right!
Let me update the codes!

Reply:
Done!

@liamHowatt Sure! I've updated the codes now. https://github.com/lvgl/lvgl/pull/5890#discussion_r1530297242